### ISSUE TITLE
[A11y] enhance text color contrast

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#170697]
### Use case

WCAG publicshed a level AAA text color contrast that raise to 7:1. 

The visual presentation of [text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced#dfn-text) and [images of text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced#dfn-image-of-text) has a [contrast ratio](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced#dfn-contrast-ratio) of at least 7:1, except for the following:

Large Text
[Large-scale](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced#dfn-large-scale) text and images of large-scale text have a contrast ratio of at least 4.5:1;

Incidental
Text or images of text that are part of an inactive [user interface component](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced#dfn-user-interface-component), that are [pure decoration](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

Logotypes
Text that is part of a logo or brand name has no contrast requirement.

### Proposal

Our current `MinimumTextContrastGuideline` targets 4.5:1 for normal text and 3:1 for large text. We should add a variant of this test that target  7:1 for normal text and 4.5:1 for large text

**Comment by azatech** [comment]
@chunhtai Created new PR. Happy to make any changes if needed!

**Comment by azatech** [comment]
@chunhtai Done. Thank you!

**Comment by Renzo-Olivares** [comment]
Can this be closed now that https://github.com/flutter/flutter/pull/170758 is merged? or is there remaining work here.

**Comment by azatech** [comment]
Yes! It can be closed, I've done everything on this one.
